app/common/translate.py

```diff
@@ -220,13 +274,18 @@ def sanitize_and_translate(self, text: str, wrap_width: int, max_lines=None, add
         for alignment in alignments:
             output = output.replace(alignment, "")
 
+        # trim multiple ellipses to a single one
+        ellipses = ["…………………………………………", "………………………………………", "……………………………………", "…………………………………", "………………………………", "……………………………", "…………………………", "………………………", "……………………", "…………………", "………………", "……………", "…………", "………", "……"]
```

Comment:
This is a long list -- can you format this more like:

```python
ellipses = [
  "…………………………………………",
  "………………………………………",
  ..
  ..
]
```

Reply:
Yep